### PR TITLE
Restore missing && to test, correct expectation.

### PR DESCRIPTION
**Pull request by mcovarr**
staring at test failures on branch leads to realization that test was broken on develop

### CODE REVIEW

**Comment by mcovarr** [comment]
@Horneth / @geoffjentry 2016
Make Cromwell Great Again

**Comment by Horneth** [comment]
:+1:

File: src/test/scala/cromwell/webservice/CromwellApiServiceSpec.scala [hunk 1]
```diff
@@ -379,7 +379,7 @@ class CromwellApiServiceSpec extends FlatSpec with CromwellApiService with Scala
           fields.get("status").isDefined &&
           fields.get("status").get.asInstanceOf[JsString].value.equals("fail") &&
           fields.get("message").isDefined &&
-          fields.get("message").get.asInstanceOf[JsString].value.contains("failed to process inputs")
+          fields.get("message").get.asInstanceOf[JsString].value.contains("Workflow input processing failed.") &&
```
Comment (cjllanwarne) [inline src/test/scala/cromwell/webservice/CromwellApiServiceSpec.scala]: Hey! I just re-added this like 2 months ago! Do we have an infiltrator with a campaign of && removal!?

**Comment by geoffjentry** [comment]
Is this "waiting for review" or "in review"?

**Comment by geoffjentry** [comment]
nm 'waiting for review' is an artifact of waffle atm

**Comment by geoffjentry** [comment]
:+1:

**Pull request merged**